lib/fabrik/DockerInstance.js

```diff
@@ -82,7 +91,9 @@ class DockerInstance extends BaseInstance {
   delete(params) {
     /* jshint unused:false */
     return this
-      .deleteSecurityGroup()
+      .platformManager.preInstanceDeleteOperations({
+        ruleId: this.securityGroupName
```

Comment:
security group name can be within platform manager. no need to pass this here.

Reply:
Removed